spec/acceptance/puppetwebhook_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper_acceptance'
+
+describe 'puppetwebhook' do
+  it 'applies' do
+    pp = <<-MANIFEST
+    class { 'puppetwebhook': }
+    MANIFEST
+
+    apply_manifest(pp, catch_failures: true)
```

Comment:
we should always run it twice, to check for idempotence?

Reply:
It will eventually. That was there for validating that the `to_yaml()` function did what was expected. I'm not sure I'm ready to test for idempotency. That will come once the service class is done